rust/otap-dataflow/crates/engine/src/pipeline_ctrl.rs

```diff
@@ -186,14 +226,19 @@ impl<PData> PipelineCtrlMsgManager<PData> {
                         PipelineControlMsg::CancelTelemetryTimer { node_id, _temp } => {
                             self.telemetry_timers.cancel(node_id);
                         }
+            PipelineControlMsg::DelayData { node_id, when, data } => {
+                let delayed = Delayed { node_id, when, data };
+                self.delayed_data.push(delayed);
+            }
```

Comment:
`cargo fmt` not correcting the indentation inside the macro automatically

```suggestion
                    PipelineControlMsg::DelayData { node_id, when, data } => {
                        let delayed = Delayed { node_id, when, data };
                        self.delayed_data.push(delayed);
                    }
```